.github/workflows/clean-tsv.yml

```diff
@@ -11,7 +11,7 @@ on:
       - source-data/gisaid_annotations.tsv
 
 jobs:
-  locations:
+  clean-metadata:
```

Comment:
Can you fixup the changes in this commit to the previous commit? It looks like this commit is just edits that belong in the prior one.